### PR TITLE
(fixes #201) ellipsis long category text overflow

### LINKED ISSUE #201: long string key values cause unfortunate overflows
Kapture 2020-01-17 at 17 08 20

### PR DESCRIPTION
**Pull request by spasovski**
closes #201 

image

### CODE REVIEW

File: src/components/explore/ComparisonSummary.svelte [hunk 1]
```diff
@@ -198,7 +198,7 @@ text: 'compares the numeric values of the reference ⭑ to the hovered values 
     <tbody>
           {#each displayValues as {leftValue, rightValue, percentageChange, key}}
             <tr>
-              <td  style="width: max-content" class=value-label>
+              <td class="value-label" title={keyFormatter(key)}>
```
Comment (hamilton) [inline src/components/explore/ComparisonSummary.svelte]: wondering if it'd be good to add `use:tooltip` on these TDs:

```
<script>
import { tooltip } from 'udgl/utils/tooltip';
...
</script>

... 
<td class="value-label" use:tooltip={{text: keyFormatter(key), location: 'top'}}>
```

File: src/components/explore/ComparisonSummary.svelte [hunk 2]
```diff
@@ -198,7 +199,7 @@ text: 'compares the numeric values of the reference ⭑ to the hovered values 
     <tbody>
           {#each displayValues as {leftValue, rightValue, percentageChange, key}}
             <tr>
-              <td  style="width: max-content" class=value-label>
+              <td class="value-label" use:tooltipAction={{text: keyFormatter(key), location: 'top'}}>
```
Comment (hamilton) [inline src/components/explore/ComparisonSummary.svelte:202]: if I hover over the other graph, it appears the tooltip disappears

**Review by hamilton — CHANGES_REQUESTED** [review]
Looking good @spasovski – noticed one minor bug. Once that's fixed, feel free to merge

**Comment by spasovski** [comment]
Will reopen a new PR for this.

**Pull request closed without merging**